app/src/main/java/org/fossasia/openevent/app/data/db/DatabaseRepository.java

```diff
@@ -0,0 +1,36 @@
+package org.fossasia.openevent.app.data.db;
+
+import com.raizlabs.android.dbflow.rx2.language.RXSQLite;
+import com.raizlabs.android.dbflow.sql.language.SQLOperator;
+import com.raizlabs.android.dbflow.sql.language.SQLite;
+import com.raizlabs.android.dbflow.structure.BaseModel;
+
+import org.fossasia.openevent.app.data.db.contract.IDatabaseRepository;
+
+import io.reactivex.Completable;
+import io.reactivex.Observable;
+
+public class DatabaseRepository implements IDatabaseRepository {
```

Comment:
Please see corresponding tests for `shouldUpdateEventsOnReload` and `shouldUpdateEventOnReload` and `getAbstractObservable`

The basic idea of tests is if these are failing, the functionality doesn't work, but if they are passing, it definitely works